runtime-services-modeling/deploying-services.adoc

```diff
@@ -64,852 +114,121 @@ $ java -jar target/quarkus-app/quarkus-run.jar
---
-. Open or import the project in your VSCode IDE to view the contents.
-
-[id="proc-kogito-creating-project-custom"]
-=== Creating a custom {PRODUCT} project using code scaffolding
-
-WARNING: This feature is experimental and might be substantially modified or removed in a future release. This feature also requires a special project setup. The generated code uses APIs that are not final in {PRODUCT} and will not function the same way in future releases.
-
-[role="_abstract"]
-The Quarkus extension and Maven plug-in within a standard {PRODUCT} project generate all the required code and boilerplate for your {PRODUCT} services so that you do not have to write the project setup code yourself. By default, a {PRODUCT} project generates a REST service from business assets automatically. The generated service usually exposes default REST endpoints using the information that is inferred from the business assets that you include in your project.
-
-If you want to customize the code for your {PRODUCT} project, such as adding more REST endpoints, you can use _code scaffolding_ to generate the project code to a specified user directory and then modify the code as needed.
-
-With code scaffolding, you generate your {PRODUCT} project with the {PRODUCT} Maven archetype as usual, but you disable the project ability to automatically regenerate the project code and explicitly instruct {PRODUCT} to generate the code in a custom target directory. The code is generated one time in the specified target directory and is not regenerated at build time unless you explicitly request it. You can customize the project code in the specified target directory as needed.
-
-NOTE: On Quarkus, disabling code regeneration also disables hot reload of business assets in development mode. This limitation will be improved in a future release.
-
-.Procedure
-. In a command terminal, navigate to a local folder where you want to store the custom {PRODUCT} project and enter the following command to generate your {PRODUCT} project with the {PRODUCT} Maven archetype as usual:
-+
---
-.On Quarkus
-[source,subs="attributes+"]
-----
-$ mvn io.quarkus:quarkus-maven-plugin:create \
-    -DprojectGroupId=org.acme -DprojectArtifactId=sample-kogito \
-    -DprojectVersion=1.0.0-SNAPSHOT -Dextensions=kogito-quarkus
-----
-
-Alternatively, you can generate a project using the https://code.quarkus.io/?e=org.kie.kogito%3Akogito-quarkus[Quarkus website].
-
-////
-@comment: The following standard command for Quarkus isn't working currently but may be restored for Dev Preview
-
-mvn io.quarkus:quarkus-maven-plugin:create -DprojectGroupId=com.company -DprojectArtifactId=sample-kogito -Dextensions="kogito"
-////
+. Run {CANVAS} - see the xref:installation/canvas.adoc[installation] chapter for more details on how to install and run it
```

Comment:
These xrefs won't work, you need to go up one level first. xref:../installation/canvas.adoc[installation]

Reply:
xrefs update, thanks